tests/src/readytorun/r2rdump/files/MultipleRuntimeFunctions/MultipleRuntimeFunctions.cs

```diff
@@ -6,16 +6,21 @@ namespace MultipleRuntimeFunctions
 {
     class MultipleRuntimeFunctions
     {
-        static void Main(string[] args)
-        {
-            try
+		static void MethodWithMultipleRuntimeFunctions()
```

Comment:
Nit: We use the [CoreFX coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md). Please can you ensure we use spaces instead of tabs for indentation.